docs/user-guide/testing-models.rst

```diff
@@ -188,9 +250,9 @@ near [Alexanderplatz](loc) [tonight](time)          O   loc time (2)          O
 ==================================================  ========================  ===========================
 
 
-.. _core-evaluation:
+.. _core-testing:
 
-Evaluating a Core Model
+Testing a Core Model
```

Comment:
@TyDunn not too sure about this one - thoughts on whether it should be testing or evaluating? Evaluating sounds way more natural to me tbh